lib/dnsimple/client/domains_email_forwards.rb

```diff
@@ -0,0 +1,86 @@
+module Dnsimple
+  class Client
+    module DomainsEmailForwards
+
+      # Lists the email forwards for the domain.
+      #
+      # @see http://developer.dnsimple.com/v2/domains/email-forwards/#list
+      #
+      # @example List email forwards in the first page
+      #   client.domains.email_forwards(1010, "example.com")
+      #
+      # @example List email forwards, provide a specific page
+      #   client.domains.email_forwards(1010, "example.com", query: { page: 2 })
+      #
+      # @param  [Fixnum, Dnsimple::Client::WILDCARD_ACCOUNT] account_id the account ID or wildcard
+      # @param  [#to_s] domain_id The domain id or domain name
+      # @param  [Hash] options the filtering and sorting option
+      # @return [Dnsimple::PaginatedResponse<Dnsimple::Struct::EmailForward>]
+      #
+      # @raise  [Dnsimple::RequestError]
```

Comment:
Both: `Request::Error` in case of a request issue, `Dnsimple::NotFoundError` if the parent resource (in this case the domain) is not found.